### PR TITLE
DDCYLS-4505 - EORI Checker (BE): check-eori-number Upgrade to 'com.typesafe.play.play'

### PR DESCRIPTION
**Pull request by tapiwa-tiyemba**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
+val silencerVersion = "1.7.16"
+
+lazy val it = project
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test") // the "test->test" allows reusing test code and test dependencies
+  .settings(DefaultBuildSettings.itSettings())
+  .settings(libraryDependencies ++= AppDependencies.test)
+
+lazy val scoverageSettings: Seq[Setting[_]] = Seq(
+  coverageExcludedPackages := "<empty>;Reverse.*;.*config.ErrorHandler;.*components.*;" +
```
Comment (ashwinirudrappa) [inline build.sbt:32]: @tapiwa-tiyemba For Backend code will not need viewModels, components, javascript, views and all,

File: it/test/connectors/EISConnectorISpec.scala [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      .configure(
+        Map("microservice.services.eis.port" -> mockServer.port(), "microservice.services.eis.environment" -> "IT")
+      )
+      .build()
+
+    lazy val connector = app.injector.instanceOf[EISConnectorImpl]
+
+    "EISConnector" - {
+
+      "return eori check response" in {
```
Comment (ashwinirudrappa) [inline it/test/connectors/EISConnectorISpec.scala:58]: @tapiwa-tiyemba Can we add a test to check the  negative scenario

**Review by ashwinirudrappa — APPROVED** [review]
